src/components/multilineBox/examples/MultilineBoxExamples.tsx

```diff
@@ -23,6 +24,43 @@ const MultilineBoxWithContainer = _.compose(
     }),
 )(MultilineBox);
 
+const MultilineBoxWithRemoveButton = _.compose(
+    multilineBoxWithRemoveButton(),
+)(MultilineBox);
+
+const MultilineBoxWithContainerAndRemoveButton = _.compose(
+    multilineBoxWithRemoveButton(),
+    multilineBoxContainer({
+        container: {
+            tag: 'div',
+            props: {
+                className: 'p1',
+            },
+        },
+    }),
+)(MultilineBox);
+
+const MultilineBoxWithContainerAndTwoRemoveButton = _.compose(
+    multilineBoxWithRemoveButton(),
+    multilineBoxContainer({
+        container: {
+            tag: 'div',
+            props: {
+                className: 'mod-border p1',
+            },
+        },
+    }),
+    multilineBoxWithRemoveButton(),

```

Comment:
You could do the same thing for the remove button (but with a default value that render your actual svg)